components/studio/build-docker-image.ps1

```diff
@@ -23,14 +23,23 @@ try {
     $env:HAB_BINLINK_DIR = $null
     
     info "Installing and extracting initial Habitat packages"
+    # Generally, we want to use packages that are aligned with the channel
+    # set when building the image which is usually unstable or an rc channel
+    # in our pipelines. Because the windows-service package is maintained
+    # in another repo and on a slower deployment cadence
```

Comment:
Looks like you left off the last bit of this sentence.

Reply:
oopsies. added it